docs/config.rst

```diff
@@ -1883,6 +2007,16 @@ Valid preset color variables are:
 
    .. versionadded:: 2.1
 
+.. attribute:: LP_COLOR_SCLS
+   :type: string
+   :value: $CYAN
```

Comment:
This isn't entirely true, as it actually defaults to `LP_COLOR_VIRTUALENV`. Somehow we should say that it defaults to `LP_COLOR_VIRTUALENV`, but also mention that that defaults to `CYAN`

Reply:
I was wondering whether this was a good idea to have dependencies between segments configurations. As far as I recall, this is the only occurrence, so maybe we should consider set it to CYAN and be done with it, to stay consistent?